### PR TITLE
update copyright statements to Cisco

### PR DESCRIPTION
**Pull request by mihaialexandrescu**
| Q               | A      |
| --------------- | ------ |
| Bug fix?        | no |
| New feature?    | no |
| API breaks?     | no |
| Deprecations?   | no |
| License         | Apache 2.0 |

### What's in this PR?
Updating copyright statements to Cisco Systems branding.
* .go files that previously had a Copyright Banzai statement  (mass replace with sed)
* scripts/check-header.sh script  (ran it after changes and got no error or output)
* 2 CopyrightManager files (under the `.idea` dir) that I'm not so sure about (please advise during review)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ When you are opening a PR to Koperator the first time we will require you to sig
 
 ## License
 
-Copyright (c) 2019 [Banzai Cloud, Inc.](https://banzaicloud.com)
+Copyright (c) 2019 [Cisco Systems, Inc.](https://www.cisco.com)
```
Comment (panyuenlau) [inline README.md:76]: I would change this to the following as well
```suggestion
Copyright (c) 2019 [Cisco Systems, Inc.](https://www.cisco.com) and/or its affiliates
```

Comment (mihaialexandrescu) [reply README.md]: done. thanks.

**Review by panyuenlau — APPROVED** [review]

**Review by pregnor — APPROVED** [review]
LGTM

**Comment by bartam1** [comment]
Is it ok to have both? 
https://github.com/banzaicloud/koperator/blob/f77e9f7365ca2f7054168cf84ef5c74b84022887/NOTICE#L1-L2

**Comment by mihaialexandrescu** [comment]
> Is it ok to have both?
> 
> https://github.com/banzaicloud/koperator/blob/f77e9f7365ca2f7054168cf84ef5c74b84022887/NOTICE#L1-L2

I have no idea.  Found it odd myself but the diff in this case is the ending of the second line, not its existence.

Do other reviewers have some thoughts on this ?

**Comment by pregnor** [comment]
> Do other reviewers have some thoughts on this ?

IANAL, but I think it is okay to have both, but also okay to have the Cisco one with the original date as legally Cisco took over the Banzai Cloud copyrights at the acquisition, we can update that if we want to only have Cisco with the original date or we can leave it this way, Git history will preserve the states anyway.

**Pull request merged**